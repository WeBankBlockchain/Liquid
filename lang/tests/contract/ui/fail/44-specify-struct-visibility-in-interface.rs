use liquid_lang as liquid;

#[liquid::interface(name = auto)]
mod foo {
    pub struct Baz {
        a: i32,
    }

    extern "liquid" {
        fn bar() -> u32;
    }
}

fn main() {}
