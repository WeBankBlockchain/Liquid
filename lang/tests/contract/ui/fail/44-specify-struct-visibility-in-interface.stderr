error: `struct` in interface must have no visibility modifier
 --> $DIR/44-specify-struct-visibility-in-interface.rs:5:5
  |
5 |     pub struct Baz {
  |     ^^^
